integration/k8s/trace_exec_test.go

```diff
@@ -29,9 +29,10 @@ func TestTraceExec(t *testing.T) {
 	t.Parallel()
 	ns := GenerateTestNamespaceName("test-trace-exec")
 
-	cmd := "cp /bin/date /date ; setuidgid 1000:1111 sh -c 'while true; do /date ; /bin/sleep 0.1; done'"
+	innerCmd := "while true ; do /bin/sleep 0.1 ; done"
+	cmd := fmt.Sprintf("cp /bin/sh /usr/bin/sh ; setuidgid 1000:1111 /usr/bin/sh -c '%s'", innerCmd)
```

Comment:
@alban changed the test as suggested to make sure we test both upper layer and parent upper layer fields.